javascript/src/js/requests.txt

```diff
@@ -0,0 +1,9 @@
+/json/rdm/section_info
```

Comment:
Ah gotcha, can you rename the file to something more obvious then, or stick it in the readme.